Tests/scripts/update_conf_json.py

```diff
@@ -114,14 +114,15 @@ def generate_pack_tests_configuration(pack_name, existing_test_playbooks):
     return pack_integrations, pack_test_playbooks, pack_name
 
 
-def update_new_conf_json(future):
+def update_new_conf_json(future, pack_name):
```

Comment:
```suggestion
def update_new_conf_json(future):
```
This won't work either, The process pull calls this method and it calls it only with the future as argument

Reply:
Can't we get the argument from the process?

Reply:
Don't we have a way to get the pack name?

Reply:
Not here
But if its something other then timeout - it will raise exception in the processing itself and it will be more verbose